File: Java/SinglyList/Node.java
/**
 * Node
 */
public class Node<T> extends Object {

    public T data;
    public Node<T> next;

    public Node(T data, Node<T> next) {
        this.data = data;
        this.next = next;
    }

    public Node() {
        this(null, null);
    }

    public static void main(String[] args) {
        System.out.println("hello");
    }


}

File: Python/Node/node.py
class Node:
    def __init__(self, data=None):
        self.data = data
        self.next = None


n1 = Node('eggs')
n2 = Node('ham')
n3 = Node('spam')
n1.next = n2
n2.next = n3

current = n1
while current:
    print(current.data)
    current = current.next
